apps/api/src/persistence/migrations/migration-record.ts

```diff
@@ -0,0 +1,30 @@
+import BaseDomainModel from '../../domain/models/BaseDomainModel';
+import cloneToPlainObject from '../../lib/utilities/cloneToPlainObject';
+import { ICoscradMigration } from './coscrad-migration.interface';
+import { CoscradMigrationMetadata } from './decorators';
+
+export class ArangoMigrationRecord extends BaseDomainModel {
```

Comment:
Do we really want to make this extend base domain model? I know we just want a `toDTO` method, but this has nothing to do with the domain and could potentially live in a different lib.